docker/observability/prometheus/rules/zebra_alerts.yml

```diff
@@ -0,0 +1,60 @@
+# Zebra Alerting Rules
+#
+# These rules are evaluated by Prometheus and sent to AlertManager.
+# AlertManager handles routing, grouping, and notifications.
+#
+# To receive notifications, configure receivers in:
+#   docker/observability/alertmanager/alertmanager.yml
+
+groups:
+  - name: zebra
+    rules:
+      # Node is down or metrics endpoint unreachable
+      - alert: ZebraDown
+        expr: up{job="zebra"} == 0
+        for: 2m
+        labels:
+          severity: critical
+        annotations:
+          summary: "Zebra node is down"
```

Comment:
Could there be a situation where the metrics endpoint is unreachable but Zebra is working as normal?